test/models/multiple-results-event.tests.js

```diff
@@ -0,0 +1,427 @@
+      input.resultSetStartTime = {id:2};
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetStartTime must be a Number");
+    });
+
+    it('It should throw if betStartTime is null', () => {
+      input.betStartTime = null;
+      assert.throws(() => new MultipleResultsEvent(input), Error, "betStartTime must be a Number");
+    });
+
+    it('It should throw if betStartTime is number', () => {
+      input.betStartTime = "12";
+      assert.throws(() => new MultipleResultsEvent(input), Error, "betStartTime must be a Number");
+    });
+
+    it('It should throw if betStartTime is array', () => {
+      input.betStartTime = [2];
+      assert.throws(() => new MultipleResultsEvent(input), Error, "betStartTime must be a Number");
+    });
+
+    it('It should throw if betStartTime is object', () => {
+      input.betStartTime = {id:2};
+      assert.throws(() => new MultipleResultsEvent(input), Error, "betStartTime must be a Number");
+    });
+
+    it('It should throw if resultSetEndTime is null', () => {
+      input.resultSetEndTime = null;
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetEndTime must be a Number");
+    });
+
+    it('It should throw if resultSetEndTime is number', () => {
+      input.resultSetEndTime = "12";
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetEndTime must be a Number");
+    });
+
+    it('It should throw if resultSetEndTime is array', () => {
+      input.resultSetEndTime = [2];
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetEndTime must be a Number");
+    });
+
+    it('It should throw if resultSetEndTime is object', () => {
+      input.resultSetEndTime = {id:2};
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetEndTime must be a Number");
+    });
+  });
+
+  describe('format', () => {
+    let input;
+    const results = ['Invalid', '1', 'y'];
+
+    beforeEach(() => {
+      input = {
+        txid: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428",
+        txStatus: TX_STATUS.SUCCESS,
+        blockNum: 5,
+        address: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428",
+        ownerAddress: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428",
+        version: 6,
+        name: "Test",
+        numOfResults: 2,
+        centralizedOracle: "0xd5d087daabc73fc6cc5d9c1131b93acbd53a2428",
+        betStartTime: 1560965704,
+        betEndTime: 1560965704,
+        resultSetStartTime: 1560965704,
+        resultSetEndTime: 1560965704,
+        escrowAmount: "1000000",
+        arbitrationLength: 172800,
+        thresholdPercentIncrease: "10",
+        arbitrationRewardPercentage: 10,
+        consensusThreshold: "100000000000",
+        arbitrationEndTime: 1560965704,
+      };
+
+      input.results = filter(
+        map(results, (item) => {
+          try {
+            return web3.utils.utf8ToHex(item);
+          } catch (err) {
+            return '(parse error)';
```

Comment:
dont really need this logic here since were not inputting bad results for the tests